Project.toml

```diff
@@ -0,0 +1,20 @@
+name = "ImageDraw"
+uuid = "4381153b-2b60-58ae-a1ba-fd683676385f"
+version = "0.2.0"
+
+[deps]
+Distances = "b4f34e82-e78d-54a5-968a-f98e89d6e8f7"
+ImageCore = "a09fc81d-aa75-5fe9-8630-4744c3626534"
+
+[compat]
+Distances = ">= 0.1.0"
+ImageCore = ">= 0.8.1"
+julia = ">= 1.0"
```

Comment:
As I've learned myself recently, it may be better to use `julia = "1"`. The best guess is that Julia 2.0.0 will break this package in some way until it gets modified for whatever language changed that introduces, so better to declare incompatibility right away.